runtime/gc_structs/ClassIteratorClassSlots.cpp

```diff
@@ -71,7 +71,14 @@ GC_ClassIteratorClassSlots::nextSlot()
 			return slotPtr;
 		}
 		_state += 1;
-		
+	
+	case classiteratorclassslots_state_default:
+		_state += 1;
```

Comment:
State should be changed after if statement, not here

Reply:
I tried to place it after the if statement at the beginning, but it looks like it falls into an infinite loop in that way.